app/presenters/GroupPresenter.php

```
@@ -0,0 +1,123 @@
+    {
+        $this->template->title = 'Create Group';
+        $this['groupForm']['groupForm']
+            ->addSubmit('submit', 'Create')
+            ->getControlPrototype()
+            ->addClass('btn btn-success');
+    }
+
+    /**
+     * @param int $id
+     */
+    public function renderEdit($id)
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)